go/grpcweb/wrapper.go

```diff
@@ -123,6 +123,15 @@ func (w *WrappedGrpcServer) ServeHTTP(resp http.ResponseWriter, req *http.Reques
 		resp.WriteHeader(http.StatusForbidden)
 		_, _ = resp.Write(make([]byte, 0))
 		return
+	} else if w.opts.enableWebsocketChannels && w.IsGrpcWebSocketChannelRequest(req) {
+		if w.websocketOriginFunc(req) {
+			if !w.opts.corsForRegisteredEndpointsOnly || w.isRequestForRegisteredEndpoint(req) {
+				w.HandleGrpcWebsocketRequest(resp, req)
```

Comment:
This should be `HandleGrpcWebsocketChannelRequest` right?

Reply:
Good catch.